### PR TITLE
- Splite stacked documentation to parts and move non releated stuff

### PR DESCRIPTION
**Pull request by eb-so**
- Use dropdowns instead of naked files to enhance the heirachy and flow

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -14,18 +14,18 @@
...
-   */
+  // /*
+  // tutorialSidebar: [
+  //   {
+  //     type: 'category',
+  //     label: 'Tutorial',
+  //     items: ['hello'],
+  //   },
+  // ],
+  //  */
```
Comment (ferrarafer) [inline sidebars.js:28]: This can be removed, right?

**Review by ferrarafer — COMMENTED** [review]
LGTM

File: sidebars.js (hunk 1 above)
Comment (FilledStacks) [reply sidebars.js:28]: @ferrarafer yes, but none of us know the framework well enough to figure out how to add it back lol

**Review by FilledStacks — APPROVED** [review]
This is much better than what I expected. Thank you for breaking our readme into the proper structure. That's very helpful. 

I'll merge it in and it'll get published after it passes a build

File: sidebars.js (hunk 1 above)
Comment (eb-so) [reply sidebars.js:28]: @ferrarafer It was already commented by the framework but I double commented it by accident

**Comment by eb-so** [comment]
@FilledStacks I'm glad you like it, However, It needs more work to smooth the flow of the reading experience

**Pull request merged**